src/scenes/Partners/Detail/PartnerDetail.tsx

```diff
@@ -205,7 +205,7 @@ const PartnerTabs = (props: PartnerViewEditProps) => {
         <Tab label="Engagements" value="engagements" />
         <Tab label="Notes" value="notes" />
       </TabList>
-      <Paper sx={{ maxWidth: 'lg' }}>
+      <Paper sx={{ maxWidth: filters.tab === 'engagements' ? 'auto' : 'lg' }}>
```

Comment:
I'd rather not do this. It breaks the open/close principle.
Can we do
```ts
<Paper sx={(theme) => ({
  '--maxWidth': theme.breakpoints.values.lg,
  maxWidth: 'var(--maxWidth)',
}}
```
And in the engagements tab
```ts
sx={{
  ':root': {
    '--maxWidth': 'initial',
  }
}}
```